app/models/video.py

```diff
@@ -1,4 +1,18 @@
+from sqlalchemy.orm import backref
 from app import db
 
 class Video(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(30))
+    release_date = db.Column(db.Date)
+    total_inventory = db.Column(db.Integer)
+    # customers = db.relationship('Customer', secondary='rentals', backref='videos')
```

Comment:
Why no rental relationship?

Reply:
From examples we looked at, I believe we just didn't realize it would be needed/useful. Could you explain why it would be a good idea?

Also, if I remember correctly, we were having issues when we tried to implement a relationship to customers. When trying to debug, we decided to deactivate that part of our code, and it went back to working. Was there a way to get that to work?

Reply:
You can setup a relationship in Video like this.  It will also populate the Rental model with a `.video` attribute

```python
rentals = db.relationship('Rental',backref='video', lazy=True)
```